src/components/assessments/AssessmentImage.tsx

```diff
@@ -14,12 +15,13 @@ const CardTop = styled(Box, {label: 'StyledCardMedia'})(({theme}) => ({
   flexDirection: 'row',
   '& img': {
     width: '150px',
-    height: '150px',
+    // height: '150px',
     backgroundColor: '#f6f6f6', //ALINA TODO PROD CHANGE - remove when images updated to new
```

Comment:
Is this comment still relevant?

Reply:
Turns out it is, in that the images are updated to *not* include a background color.